### PR TITLE
[TRAFODION-2267] Fix ALTER TABLE RENAME documentation

### PR DESCRIPTION
**Pull request by DaveBirdsall**
Problem: The SQL Reference manual describes a CASCADE keyword for ALTER TABLE RENAME, however if you try to use it, you get error message 1427, "Table cannot be renamed. Reason: Cascade option not supported."

Supporting this option seems like a fair amount of work, so for the moment, we've simply removed mention of CASCADE from the SQL reference manual.

While working on this issue, I noticed that the DDL messages section of the Messages Guide is woefully out-of-date. Among other things, it lacks several messages including the message 1427 mentioned above. Fixing all of that is a big job, but with this set of changes I did a small part of it. I added documentation for messages 1420-1429.

While doing this I noticed that ALTER TABLE RENAME to an invalid HBase name produced a very ugly 8448 error (unlike CREATE TABLE and DROP TABLE which produce a nice tidy error 1422 in this case). I fixed the error reporting code in ALTER TABLE RENAME so that we now get the nice error 1422 instead.

### CODE REVIEW

File: docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc [hunk 1]
```diff
@@ -3044,3 +3044,146 @@ statement.
...
+[[SQL-1422]]
+== SQL 1422 
+
+```
+An invalid HBase name was specified in this DDL statement. A valid name can only contain these characters: [a-zA-Z_0-9-.]
+```
+
+*Cause:* You attempted to use a name containing an invalid character for a {project-name} object that is
+stored in HBase.
+
```
Comment (robertamarton) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: I assume {project-name} gets changed to Trafodion during document rendering.

File: docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc [hunk 2]
```diff
@@ -3044,3 +3044,146 @@ statement.
...
+*Recovery:* None. Contact the {project-name} Developer Distribution List.
+
+<<<
+[[SQL-1429]]
+== SQL 1429
+
+```
+Inserts into _ROW_ format external hbase tables can only use the VALUES clause and must use the column_create function to create values.
+```
```
Comment (robertamarton) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: hbase ->HBase?

Comment (DaveBirdsall) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: Oops... I'll fix this.

File: docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc [hunk 3]
```diff
@@ -3044,3 +3044,146 @@ statement.
...
+[[SQL-1427]]
+== SQL 1427 
+
+```
+Table cannot be renamed. <reason>
+```
+
+*Cause:* You attempted to perform ALTER TABLE RENAME but the rename could not be performed.
+The <reason> explains why.
+
```
Comment (robertamarton) [inline docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc]: nie - In this Cause - "The <reason> explains why." is used.  In message 1425 "<reason> gives more details", may want to make both the same.

**Comment by robertamarton** [comment]
I personally, forget to change the messages manually when a new message is added/updated.  Perhaps as part of reviews, we should verify that this step is taken.
Is there a JIRA to support the CASCADE option at some point in time; that is, is this something that would be needed to make names more meaningful?

Changes look fine, just a couple of nits.

**Comment by DaveBirdsall** [comment]
@robertamarton, there's no JIRA to support CASCADE at the moment. It looks like intent behind CASCADE is for the rename to succeed even if dependent objects exist. (If you try a rename now and a view exists, the operation fails with error 1427, "Table cannot be renamed. Reason: Operation not allowed if dependent views exist. Drop the views and recreate them after rename." I can't be completely sure that this is the whole intent, so some definitional work needs to be done before such a JIRA can be written.

**Pull request merged**